### PR TITLE
add vsphere and mysql_mariadb template for influxdb v2.0

### PR DESCRIPTION
**Pull request by xe-nvdk**

### CODE REVIEW

File: tomcat/telegraf.conf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  ## Destination bucket to write into.
+  bucket = "tomcat"
+
+
+
+ # Gather metrics from the Tomcat server status page.
+  [[inputs.tomcat]]
+  ## URL of the Tomcat server status
+  url = "http://127.0.0.1:8888/manager/status/all?XML=true"
```
Comment (mhall119) [inline tomcat/telegraf.conf:23]: Make an environment variable

File: tomcat/telegraf.conf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  ## HTTP Basic Auth Credentials
+  username = "tomcat"
```
Comment (mhall119) [inline tomcat/telegraf.conf:26]: Make an environment variable

File: tomcat/telegraf.conf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  password = "tomcat"
```
Comment (mhall119) [inline tomcat/telegraf.conf:27]: Make an environment variable

File: vsphere/telegraf.conf [hunk 4]
```diff
@@ -0,0 +1,198 @@
+ [[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  ##
+  ## Multiple URLs can be specified for a single cluster, only ONE of the
+  ## urls will be written to each interval.
+  ## urls exp: http://127.0.0.1:9999
+  urls = ["http://localhost:9999"]
```
Comment (mhall119) [inline vsphere/telegraf.conf:7]: Make an environment variable

File: vsphere/telegraf.conf [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+  ## Token for authentication.
+  token = ""
+
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "data"
+
+  ## Destination bucket to write into.
```
Comment (mhall119) [inline vsphere/telegraf.conf:15]: Make an environment variable

File: vsphere/telegraf.conf [hunk 6]
```diff
@@ -0,0 +1,198 @@
+ [[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  ##
+  ## Multiple URLs can be specified for a single cluster, only ONE of the
+  ## urls will be written to each interval.
+  ## urls exp: http://127.0.0.1:9999
+  urls = ["http://localhost:9999"]
+
+  ## Token for authentication.
+  token = ""
```
Comment (mhall119) [inline vsphere/telegraf.conf:10]: Make an environment variable

File: vsphere/telegraf.conf [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "data"
```
Comment (mhall119) [inline vsphere/telegraf.conf:13]: Make an environment variable

File: vsphere/telegraf.conf [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+  bucket = "vsphere"
+
+  [agent]
+  interval = "30s"
+
+# Read metrics from one or many vCenters
+[[inputs.vsphere]]
+    ## List of vCenter URLs to be monitored. These three lines must be uncommented
+  ## and edited for the plugin to work.
+  vcenters = [ "https://192.168.76.174/sdk" ]
```
Comment (mhall119) [inline vsphere/telegraf.conf:25]: Make an environment variable

File: zookeeper/readme.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# ZooKeeper Dashboard for InfluxDB v2
```
Comment (mhall119) [inline zookeeper/readme.md:1]: This isn't in the table in the main `README` file

File: zookeeper/telegraf.conf [hunk 10]
```diff
@@ -0,0 +1,36 @@
+ [[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  ##
+  ## Multiple URLs can be specified for a single cluster, only ONE of the
+  ## urls will be written to each interval.
+  ## urls exp: http://127.0.0.1:9999
+  urls = ["http://localhost:9999"]
```
Comment (mhall119) [inline zookeeper/telegraf.conf:7]: Make an environment variable

File: zookeeper/telegraf.conf [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+  ## Token for authentication.
+  token = ""
```
Comment (mhall119) [inline zookeeper/telegraf.conf:10]: Make an environment variable

File: zookeeper/telegraf.conf [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "data"
```
Comment (mhall119) [inline zookeeper/telegraf.conf:13]: Make an environment variable

File: zookeeper/telegraf.conf [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+  ## Destination bucket to write into.
+  bucket = "zookeeper"
```
Comment (mhall119) [inline zookeeper/telegraf.conf:16]: Make an environment variable

File: zookeeper/telegraf.conf [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+# Reads 'mntr' stats from one or many zookeeper servers
+[[inputs.zookeeper]]
+  ## An array of address to gather stats about. Specify an ip or hostname
+  ## with port. ie localhost:2181, 10.0.0.1:2181, etc.
+
+  ## If no servers are specified, then localhost is used as the host.
+  ## If no port is specified, 2181 is used
+  servers = ["localhost:2181"]
```
Comment (mhall119) [inline zookeeper/telegraf.conf:25]: Make an environment variable

File: vsphere/telegraf.conf (hunk 4 above)
Comment (xe-nvdk) [reply vsphere/telegraf.conf]: I comment on the login and hosts, no only in this one, in all. But make an environment variable the name of a host is too much. I comment on the line, so, the user can see what needs to edit.

**Pull request merged**